terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/message/MessageTest.java

```diff
@@ -732,4 +732,41 @@ public void test09_02_disableHtmlEscape() {
         screenCapture.save(driver);
     }
 
+    @Test
+    public void test10_01_tagConfiguration() {
+        driver.findElement(By.id("tagConfiguration_10_01")).click();
+
+        // innerElement close with </li> confirm
+        assertThat(driver.getPageSource().contains(
+                "<div class=\"alert alert-success\"><ul><li></li></ul></div>"),
+                is(true));
+
+        // screen capture
+        screenCapture.save(driver);

```

Comment:
I think, these three cases are no difference in the appearance of the result screen.
So, it should take page source for evidence.